include/openenclave/edl/sgx/attestation.edl

```diff
@@ -23,6 +23,12 @@ enclave
     // contiguous memory.
     include "openenclave/bits/sgx/sgxtypes.h"
 
+    struct format_ids_t
+    {
+        [size=size] void* data;
```

Comment:
Given deep-copy support, is it possible to make the ocall strongly typed? 
The caller and callee cast seem to use `oe_uuid_t` for data.

Reply:
Yes, but doing so requires the oeedger8r-generated files include the `evidence.h` where `oe_uuid_t` is defined. Keeping the use of void seems easier.

Reply:
FWIW, I'd prefer including evidence.h but don't feel strongly